app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/extrabuildings/ExtraBuildingBottomSheet.kt

```diff
@@ -56,6 +62,8 @@ internal fun ExtraBuildingBottomSheet(
     shape = MaterialTheme.shapes.squircleLargeTop,
     sheetState = sheetState,
     tonalElevation = 0.dp,
+    windowInsets = WindowInsets.ime,
```

Comment:
For this btw, could a solution instead be that the content itself is the one that takes the IME insets?

Because now I think that the entire sheet will have the IME insets, and the content will *also* receive them, getting them double applied.

So the sheet probably is offset up by the IME height, and then inside the content itself there is also a space at the bottom equal to the IME height, am I correct in my assumption here?

Reply:
I'll check this out. I tried to add ime to the content in one of my attempts, but maybe it was not done correctly